eotsmanager/store/eotsstore.go

```diff
@@ -292,3 +294,37 @@ func ExtractHeightFromKey(key []byte) (uint64, error) {
 	// Use sdk.BigEndianToUint64 to convert back to uint64
 	return sdk.BigEndianToUint64(heightBytes), nil
 }
+
+// BackupDB performs a hot backup of the database using a read-only transaction
+func (s *EOTSStore) BackupDB(dbPath string, backupDir string) error {
+	if dbPath == "" {
+		return fmt.Errorf("database path must be provided")
+	}
+
+	if backupDir == "" {
+		return fmt.Errorf("backup dir path must be provided")
+	}
+
+	// Create backup filename with timestamp
+	backupName := "eots.db"
```

Comment:
Would it be better to let user specify the backup name? Maybe they want to add timestamp or something